remoteappmanager/static/js/home/configurables.js

```diff
@@ -1,85 +1,53 @@
 define([
-    "jquery",
-    "handlebars",
-    "utils"
-], function($, hb, utils) {
+    "utils",
+    "components/vue/dist/vue.min"
+], function(utils, Vue) {
     "use strict";
 
-    var view_template = hb.compile(
-        '<div class="form-group">' +
-        '<label for="resolution-model-{{index}}">Resolution</label>' +
-        '<select class="form-control" id="resolution-model-{{ index }}">' +
-        '{{#each options}}' +
-           '<option value="{{this}}">{{this}}</option>' +
-        '{{/each}}' +
-        '</div>'
-    );
+    var resolution_tag = 'resolution';
```

Comment:
make sure that you use reasonable differentiation between the configurable tag (e.g. "resolution") and the component tag (which is e.g. "resolution-component")